test/clj/cider/nrepl/middleware/test_test.clj

```diff
@@ -152,7 +152,11 @@
     (is (= "custom-output\n"
            (#'test/print-object (with-meta {:not :printed} {:type ::custom}))))
     (is (= "{:a :b, :c :d}\n"
-           (#'test/print-object {:a :b :c :d})))))
+           (#'test/print-object {:a :b :c :d})))
+    (is (= "{:a \"b\", :c \"d\"}\n"
+           (#'test/print-object {:a "b" :c "d"})))
+    (is (= "\"string\"\n"
```

Comment:
I'd add another case in which a `\n` can be seen. That makes it more even more evident that `pprint` is being used.